src/components/list-view/index.tsx

```diff
@@ -272,6 +279,20 @@ class ListView extends Component<Props, State> {
     const customFooterLoaded = showFooter && launchFooterLoaded && !hasMore; // 渲染renderLoadedText
     const footerLoading = showFooter && !launchFooterLoading && hasMore;
     const customFooterLoading = showFooter && launchFooterLoading && hasMore; // 渲染renderNoMore
+
+    const isScrollingRequired = this.targetScrollTop !== this.state.scrollTop && this.reachedTargetScrollTop === true;
+    const isDeltaYChanged = scrollDeltaY !== this.prevScrollDeltaY;
+
+    if (scrollDeltaY && (isScrollingRequired || isDeltaYChanged)) {
+      this.reachedTargetScrollTop = false;
+
+      const revisiting = this.targetScrollTop === this.state.scrollTop + scrollDeltaY;
+      this.targetScrollTop = !revisiting ? this.state.scrollTop + scrollDeltaY : this.state.scrollTop + scrollDeltaY + 1;
```

Comment:
1 is not good, it's a magic number, you should name it as const variable indicates it's function